playbooks/roles/mysql/files/edxapp.sql

```diff
@@ -88,6 +88,7 @@ VALUES
   We will set is_active = False. So we must use reset password inorder to set password and to activate the account.
 */
 
+DELETE FROM auth_userprofile where name='oxamaster';
 INSERT into auth_user
 (
   password,
```

Comment:
Thanks Satya. You are right this is a public repo. But user is not active by default. You have to reset password to activate it. But good point. I will reimplement with createuser commands. I will remove the superuser create script from this file.